packages/components/atoms/f-form-field/CHANGELOG.md

```diff
@@ -4,6 +4,21 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
 
+v2.0.0
+------------------------------
+*August 5, 2021*
+
+### Remove
+- `labelStyle` prop.
+- `Inline` and `InlineNarrow` styles.
+- `hasInputDescription` prop.
+
+### Added
+- `shouldShowLabelText` prop.
+- `labelDescription` prop.
```

Comment:
I would put this under a `Changed` section and put that you've removed the description slot in favour of a `labelDescription` prop.

Also, anything that is considered "Breaking" between versions (like this) is good to add *Breaking* to the front of, so that anyone using the component and upgrading can see what they might need to watch out for. So here I would add that before this label change and the InlineNarrow and labelStyle changes above.